### DISCUSSION TITLE
W5500 chip driver

### DISCUSSION THREAD

**Discussion by MagicStephen** [micropython/micropython#13623] [Libraries & Drivers]
Hello, i'd like to ask is there some way, how to implement all C functions from W5500 IOLibrary_driver ? I try tu implement my own firmware, but when i looked to libraries i found class WIZNET5K which consists only (regs,isconnected,active,ifconfig,status,config,send_ethernet). I would like to read packets in IPRAW mode, or socket raw mod, but i cant figure it out. Also i couldn't find any documentation for function send_ethernet. Is it impossible to build functions for chip ?

**Comment by jimmo** [comment]
@MagicStephen sorry I'm not quite sure what you're asking and how it relates to MicroPython? Are you trying to access the raw mode directly from MicroPython? That is not supported.
